store/src/java-test/com/zimbra/cs/mailbox/ContactAutoCompleteTest.java

```diff
@@ -296,5 +294,82 @@ public void autocompleteTestNonExistingContact() throws Exception {
         ContactRankings.increment(mbox.getAccountId(), Collections.singleton(new InternetAddress("noex@zimbra.com")));
         ContactAutoComplete autocomplete = new ContactAutoComplete(mbox.getAccount(), new OperationContext(mbox));
         Assert.assertEquals(0, autocomplete.query("noex", null, 10).entries.size());
-     }
+    }
+
+    @Test
+    public void autoCompleteTestForAliasesTwoResults() throws Exception {
+        Account account = Provisioning.getInstance().getAccountByName("testContAC@zimbra.com");
+        Mailbox mbox = MailboxManager.getInstance().getMailboxByAccountId(MockProvisioning.DEFAULT_ACCOUNT_ID);
+
+        ContactAutoComplete contactAutoComplete = spy(new ContactAutoComplete(account, new OperationContext(mbox)));
```

Comment:
Is there a reason for a spy or mock here? I don't see any mocked methods.

Reply:
Removed the spy here, as we are not using any mocks. Thanks for correcting this.